File: src/main/java/plus/maa/backend/controller/request/CopilotCUDRequest.java
package plus.maa.backend.controller.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CopilotCUDRequest {
    private String content;
    private String id;
}

File: src/main/java/plus/maa/backend/controller/request/copilot/CopilotCUDRequest.kt
package plus.maa.backend.controller.request.copilot;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CopilotCUDRequest {
    private String content;
    private Long id;
}
